README.md

```diff
@@ -127,6 +130,18 @@ especially if sysimages are involved. However:
 - you can add deps from within your julia session, and the folder 
   syncing will mirror them to your local project folder.
 
+##### Examples
+
+From a julia project folder containing Pluto as a dependency,
+
+```bash
+julia_pod --port-forward=1234:1234 'using Pluto; Pluto.run()'
+```
+
+will run a Pluto notebook and make `localhost:1234` forward to port
+`1234` in the pod running the notebook.
+
+`julia_pod --julia='"julia", "-t", "16"'` will launch a `julia_pod` with 16 threads.
```

Comment:
Future idea: use `--` to [specify the end of options](https://unix.stackexchange.com/a/187548) and start specifying the command to run. e.g.:

```
julia_pod -- julia -t 16
```